README.md

```diff
@@ -101,66 +91,46 @@
 
 ### Colors
 
-- [stylelint-color-format](https://www.npmjs.com/package/stylelint-color-format) - Enforce HEX, RGB or HSL colors.
 - [stylelint-gamut](https://www.npmjs.com/package/stylelint-gamut) - Enforce standard different color spaces (Pack).
 - [stylelint-no-indistinguishable-colors](https://www.npmjs.com/package/stylelint-no-indistinguishable-colors) - Disallow colors that are suspiciously close to being identical.
 
 ### File system
 
 - [stylelint-no-unresolved-module](https://www.npmjs.com/package/stylelint-no-unresolved-module) - Disallow unresolved imports (import-like or `url`).
 
-### Images
-
-- [stylelint-images](https://www.npmjs.com/package/stylelint-images) - Enforce good performance and avoid mistakes with images (Pack).
-
 ### Internationalization
 
 - [stylelint-plugin-logical-css](https://www.npmjs.com/package/stylelint-plugin-logical-css) - Enforce logical CSS properties, values and units (Pack).
-- [stylelint-use-logical-spec](https://www.npmjs.com/package/stylelint-use-logical-spec) - Enforce usage of logical properties and values.
 
-### Nesting and specificity
+### Nesting
 
-- [stylelint-at-rule-no-children](https://www.npmjs.com/package/stylelint-at-rule-no-children) - Disallow block rule declarations inside at-rules.
 - [stylelint-use-nesting](https://www.npmjs.com/package/stylelint-use-nesting) - Enforce nesting when it is possible.
 
+### Order
+
+- [stylelint-order](https://www.npmjs.com/package/stylelint-order) - Enforce a specific order for things like properties (Pack).
+
 ### Pre-processors
```

Comment:
note

too bad it's not maintained
we could have added [`stylelint-less`](https://github.com/ssivanatarajan/stylelint-less)